efficientdet/utils.py

```diff
@@ -622,13 +622,11 @@ def build_model_with_precision(pp, mm, ii, *args, **kwargs):
     inputs = tf.cast(ii, tf.bfloat16)
     with tf.tpu.bfloat16_scope():
       outputs = mm(inputs, *args, **kwargs)
-    set_precision_policy('float32')
```

Comment:
After remove 2 lines, I could train model with recompute_grad and mixed_precision.
Why set policy back to float32? Could I remove them? @mingxingtan

Reply:
Yes, please feel free to remove it. it is not necessary.